### ISSUE TITLE
Background color

### ISSUE THREAD

**Issue by v0idkr4ft** [cmatsuoka/asciiquarium#9]
I can't seem to figure out where this is set in the code. 
would like to change it to use my terminals background color
or make an argument -b to disable it.

**Comment by mlopezgva** [comment]
Yes, I would like to make it either deepblue sea or transparent (with a seabed pic as desktop background :-))

**Comment by hiringManager** [comment]
mine is set along with xcolor so I can change it live with any xrdb/xcolor script. It should also follow your terminal color settings, but idk. 

xterm -fg white -bg black

if you want it to be permanent --


```
type man xterm
create a file named(sensitive case) XTerm in your home directory with your specs

XTerm*Background: linen
XTerm*Foreground: black
XTerm*font: 9x15
XTerm*saveLines: 1000
XTerm*HiForeColor: white
XTerm*HiBackColor: #c06077
XTerm*geometry: +50+100
```

That being said, you won't be able to get a transparent background to my knowledge, but I have no idea how compositors will handle xterm bg being set to transparent.

**Comment by nothub** [comment]
i uploaded a version with working transparency (using default background color) that uses a [change](https://github.com/rwxrob/dot/blob/2c4b63f77dcb235be4e218cd8febc5533420ee54/scripts/fishies#L109) made by @rwxrob here:
https://gist.github.com/nothub/91ff81bd945eb8f1e6af08492a13ff75

**Comment by noobishply** [comment]
how do i install and use this version @nothub??

**Comment by nothub** [comment]
the same way as you would use the original. if there are problems specific to my version, please comment on the gist.
